### PR TITLE
Update to bStats v2

### PR DESCRIPTION
**Pull request by WalshyDev**
Keeping this as a draft for now as we may want to do something different and things may still change in bStats.

### CODE REVIEW

**Review by TheBusyBiscuit — DISMISSED** [review]
LGTM but we'll wait and keep it a draft for now.

**Review by TheBusyBiscuit — APPROVED** [review]

**Comment by WalshyDev** [comment]
Now the announcement has happened this is good to merge. I'll let you do any last review you may want or any last comments before you merge.

**Comment by TheBusyBiscuit** [comment]
Looks good to me, not really anything I'd have to say about it before merging.

**Pull request merged**
